backend/src/appointment/routes/account.py

```diff
@@ -52,3 +54,16 @@ def delete_account(db: Session = Depends(get_db), subscriber: Subscriber = Depen
         return data.delete_account(db, subscriber)
     except AccountDeletionException as e:
         raise HTTPException(status_code=500, detail=e.message)
+
+
+@router.get("/available-emails")
+def get_available_emails(db: Session = Depends(get_db), subscriber: Subscriber = Depends(get_subscriber)):
+    """Return the list of emails they can use within Thunderbird Appointment"""
+    google_connections = get_by_type(db, subscriber_id=subscriber.id, type=ExternalConnectionType.google)
+
+    emails = {subscriber.email, *[connection.name for connection in google_connections]} - {subscriber.preferred_email}
```

Comment:
This should ensure there's no duplicates if they already have their preferred email set as their main email, or a google account email.